entry.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = Entry;
```

Comment:
Yeah, this will require some massage. I will land as-is and keep this in mind for follow-up.